core/commands/cherry_pick.py

```diff
@@ -1,13 +1,40 @@
 import sublime
+from sublime_plugin import TextCommand
 
 from .log import GsLogCommand
 from ..git_mixins.branches import BranchesMixin
+from ...common import util
+
+
+class GsCherryPickHashCommand(TextCommand, BranchesMixin):
```

Comment:
Hmm... so actually, I'd like for the branch history stuff to be extracted from `interfaces/branch.py`.  The history view may be triggered from the branch dashboard, but that doesn't mean it can only be triggered from there, and it isn't really related to the branch dashboard directly.

If you'd like, I can extract that first, and then you can move `GsCherryPickHashCommand` to that new file.